scout/server/blueprints/variants/templates/variants/indicators.html

```diff
@@ -63,6 +63,15 @@
   {% endif %}
 {% endmacro %}
 
+{% macro dismissals_badge(variant) %}
+  {% if variant.dismissals %}
+      <span class="badge bg-secondary" style="margin-left:1px" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="top"
```

Comment:
```suggestion
      <span class="badge bg-warning" style="margin-left:1px" data-bs-toggle="tooltip" data-bs-html="true" data-bs-placement="top"
```

Reply:
Just an idea. You pay more attention if it's yellow. And perhaps gives you also the idea that you should be careful in your judgement